cpu/native/ldscripts/xfa.ld

```diff
@@ -1,6 +1,6 @@
 SECTIONS
 {
-    .data :
+    .data : SUBALIGN(0)
```

Comment:
Do we really only need this on `native`?

Reply:
hm, on my system, the tests-core unittest fails otherwise, the marker is not aligned.

Reply:
> Do we really _only_ need this on native?

ah, I think I missed the _only_, you meant we probably need this on the other architectures, right?